### PR TITLE
docs(website): added airgapped glossary piece

### PR DESCRIPTION
**Pull request by jakepage91**
Added the SEO optimised Airgapped glossary piece. 

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

File: website/glossary/15_airgapped.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Airgapped
```
Comment (pmig) [inline website/glossary/15_airgapped.mdx:2]:
```suggestion
title: What does air gapped mean? Definition, Meaning and Examples.
```

This is the title that appears in the search results,

File: website/glossary/15_airgapped.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+slug: airgapped-meaning
+description: 'Understanding \ airgapped systems, networks, and environments. Their purpose, use cases, and practical applications.'
```
Comment (pmig) [inline website/glossary/15_airgapped.mdx:4]:
```suggestion
description: 'Understanding air gapped computer systems, networks, and environments. Their purpose, use cases, and practical applications.'
```

File: website/glossary/15_airgapped.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+import DefaultCTA from '@site/src/components/cta/DefaultCTA/defaultCTA';
+
+### Airgapped Meaning
```
Comment (pmig) [inline website/glossary/15_airgapped.mdx:9]:
```suggestion
### Air gapped Meaning
```

File: website/glossary/15_airgapped.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+:::note
+You might have seen the term written in various ways, air gapped, air-gapped, or even referring to a system with an air gap. Regardless of how it's phrased, it all describes the same concept: systems or networks that are completely isolated from external internet connectivity, with the primary goal of enhancing security. This isolation prevents unauthorized access, for companies in highly regulated industries this is the only way of safeguarding sensitive data and protecting critical systems from cyber threats.
+:::
+### What does Airgapped usually refer to?
```
Comment (pmig) [inline website/glossary/15_airgapped.mdx:13]:
```suggestion
### What does Air gapped usually refer to? 
```

**Review by pmig — CHANGES_REQUESTED** [review]
I makes more sense to write "air gapped" instead of "airgapped" maybe also "airgapped vs air gapped" as an explanation :-)

File: website/glossary/15_airgapped.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+### Air Gapped Systems: How Do They Work?
+
+Air gapped systems work by being physically or logically disconnected from external networks. Data transfer in and out typically requires manual processes, like using USB drives or other removable media. In some cases, a liberal air gapped setup might allow restricted access to protected private registries, but such configurations are exceptions rather than the norm. For a system to be truly airgapped, all wireless connectivity, such as WiFi and Bluetooth, must be disabled, and physical connections to non-airgapped networks must be avoided.
+
+### How Secure Are Air Gapped Environments?
```
Comment (pmig) [inline website/glossary/15_airgapped.mdx:25]:
```suggestion
### How Secure Air Gapped Environments?
```

**Review by pmig — APPROVED** [review]

**Review by pmig — DISMISSED** [review]

**Review by kosmoz — DISMISSED** [review]

**Review by pmig — APPROVED** [review]

**Review by pmig — APPROVED** [review]

**Pull request merged**